client/components/file-picker/docs/example.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import FilePicker from 'components/file-picker';
+
+export default class FilePickers extends React.Component {
+	constructor( props ) {
+		super( props );
+		this.onSingle = this.onSingle.bind( this );
+		this.onMulti = this.onMulti.bind( this );
+	}
+
+	onSingle( files ) {
+		alert( 'Selected file: ' + JSON.stringify( files[0].name ) );
```

Comment:
You think so? For a doc example I thought it would be good to see the "results" of the user action kind of in-your-face. Gridicons uses `promt()` for example, which is similar.

We should have some standard way of displaying results in the docs (a built-in log-dump thing would be cool).